packages/appeal-reply-service-api/globalConfig.json

```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:52242/jest?","mongoDBName":"jest"}
```

Comment:
I've worked out why this is a problem in this package, but not in appeals-service-api.

In `/.gitignore`, please change:
```
// line ~171
packages/appeals-service-api/globalConfig.json
```

to:
```
packages/**/globalConfig.json
```

Reply:
Done.

Reply:
Can you delete this file as well?